# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/thin-provisioning-tools/hotfix.patch.cross
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- thin-provisioning-tools-1.0.2/Makefile.vanilla	2023-02-19 16:58:17.987090169 +0100
+++ thin-provisioning-tools-1.0.2/Makefile	2023-02-19 16:59:41.863099437 +0100
@@ -4,7 +4,7 @@
 	target/release/pdata_tools
 
 $(PDATA_TOOLS):
-	$(V) cargo build --release
+	#$(V) cargo build --release
 
 DESTDIR:=/usr
 BINDIR:=$(DESTDIR)/sbin
